docs/Installation.md

```diff
@@ -168,8 +168,16 @@ spec:
 
 If you don't manage your own server, the below providers will host Shiori for you. None are endorsed by or affiliated with the team. Support is provided by the providers.
 
+### CloudBreak
+
+[CloudBreak](https://cloudbreak.app/products/shiori?utm_medium=referral&utm_source=shiori-docs&rby=shiori-docs) offers Shiori hosting from $12/year ($1/month).  Get $3 off with coupon `SHIORI`.
+
+<a href="https://cloudbreak.app/products/shiori?utm_medium=referral&utm_source=shiori-docs&rby=shiori-docs">
+  <img src="https://cloudbreak.app/external/subscribe-button.png" alt="Subscribe on CloudBreak" width="149" height="64">
+</a>
+
 ### PikaPods
 
-[PikaPods](https://www.pikapods.com/) offers Shiori hosting from $1/month with $5 free welcome credit. EU and US regions available. Updates are applied weekly and user data backed up daily.
+[PikaPods](https://www.pikapods.com/) offers Shiori hosting from $1.20/month with $5 free welcome credit. EU and US regions available. Updates are applied weekly and user data backed up daily.
```

Comment:
I noticed $1.20 appears to be the new lowest price that PikaPods offers for any service.  I thought it would be appropriate to update it here.  This change can be backed out if you'd like.